src/point.js

```diff
@@ -1,9 +1,65 @@
 import React from 'react'
 import { useChart } from './chart'
 
-const Point = ({ x, y, children }) => {
+const Point = ({
+  x,
+  y,
+  children,
+  align = 'left',
+  verticalAlign = 'top',
+  width,
+}) => {
   const { x: _x, y: _y, pl, pr, pt, pb, apl, apr, apt, apb } = useChart()
 
+  let position, verticalPosition
+
+  if (!['left', 'right', 'center'].includes(align)) {
+    throw new Error(
+      `'${align}' is not a recognized alignment, must be left, right, or center`
+    )
+  }
+
+  if (!['top', 'bottom'].includes(verticalAlign)) {
+    throw new Error(
+      `'${verticalAlign}' is not a recognized vertical alignment, must be top or bottom`
+    )
+  }
+
+  if (align === 'center' && !width) {
```

Comment:
I think we discussed effectively defaulting `width` -- I think to `domain[1] - x`. One wrinkle with that default (if I'm remembering it correctly) is that using `align='center'` with no width on _multiple_ labels could easily result in overlapping labels, so I'm actually not so opposed to starting with the more explicit `width` requirement.

Reply:
kept it required for now (and same for the new `height` prop for vertical Alignment)